app/forms/openid_connect_authorize_form.rb

```diff
@@ -114,6 +120,10 @@ def aal
       Saml::Idp::Constants::DEFAULT_AAL_AUTHN_CONTEXT_CLASSREF
```

Comment:
This method currently uses `aal_values.sort.max`, should it use the new sorting functionality as well?